packages/react-examples/src/react-link/Link/Link.Basic.Example.tsx

```diff
@@ -10,8 +10,16 @@ export const LinkBasicExample: React.FunctionComponent = () => {
         <Link href="https://developer.microsoft.com/en-us/fluentui#/controls/web/link">
           it renders as an anchor tag.
         </Link>{' '}
-        Without an href, <Link>the link is rendered as a button</Link>. You can also use the disabled attribute to
-        create a{' '}
+        Without an href,
+        <Link
+          // eslint-disable-next-line react/jsx-no-bind
+          onClick={() => {
```

Comment:
Since this doesn't depend on anything within the component I'd prefer to move it to the top level of the file, and then you can remove the jsx-no-bind disable too.

Reply:
yeah I was thinking the same, but from docs perspective, it is usually unnecessary churn to docs reader/consumer to have context "far away"

Reply:
Fair point, it's a hard balance with our examples sometimes between demonstrating best practices`*` (since some people WILL copy-paste, causing any bad practices we demo to propagate) and making them easy to comprehend. Open to further discussion on this though. I think in this case since it's a pretty small file, having the onClick handler out of context isn't too bad. 

`*` I know it's also debatable how much of an issue inline functions actually cause. At this point we have our eslint set to allow inline functions to be passed to built-in elements, but not to components. The rationale being that there's no easy way for the consumer to know whether changing function identity for a given component/prop will cause problematic re-renders or re-computations of expensive effects/etc, so it's safest to avoid inline functions.

Reply:
>  it's a hard balance with our examples sometimes 
yup, we should author a style guide for this in the future which is gonna be the source (preferably supported by tooling) of truth for everyone.

> so it's safest to avoid inline functions.

hehe Im gonna use my quote again, measure first :), but I know what you mean.